packages/odie-client/src/components/send-message-input/index.tsx

```diff
@@ -87,6 +89,7 @@ export const OdieSendMessageButton = () => {
 						keyUpHandle={ onKeyUp }
 					/>
 					{ shouldBeDisabled && <Spinner className="odie-send-message-input-spinner" /> }
+					<AttachmentButton />
```

Comment:
WDYT if we add the `shouldUseHelpCenterExperience` check here instead of inside the component?, this would avoid unnecessary mounts

```js
{shouldUseHelpCenterExperience && <AttachmentButton /> }
```

Reply:
Pushed this change